addon/chrome/content/overlay/browser.css

```diff
@@ -234,6 +234,7 @@ so display: none !important; does not hide the button */
 .tab-progress {
   margin: 0;
   min-width: 0;
+  width: 0;
```

Comment:
Is it related to rename tab panel ?

Reply:
In fact, yes. Although it could be a issue of it own.

Rename tab can cause tab width become thinner.
Without that the progress bar will set to a fix width, which in my case is 120px, and in turn enlarged width of the tab.
Set it width to 0 will make it fit the tab width.
This may be OS related but I can only test on win10.